docs/ARCHITECTURE.md

```diff
@@ -197,11 +202,13 @@ sequenceDiagram
     participant AstarteMessageHub
     participant Astarte
     Node1->>AstarteMessageHub: send(message)
-    AstarteMessageHub->>Astarte: sendData()
-
+    alt check Node UUID metadata
+        AstarteMessageHub ->> Astarte: sendData()
+    end
```

Comment:
I don't know if this is the correct way to represent the node check, it should be another alternative with the failed response and unauthorize status.

Reply:
I changed the mermaid schemas. Let me know what do you think